Fabric/src/main/java/com/blamejared/crafttweaker/platform/FabricPlatformHelper.java

```diff
@@ -154,14 +171,18 @@ public Fluid getBucketContent(BucketItem item) {
         List<Mod> mods = new ArrayList<>();
         // This doesn't work for the current mod in dev.
         // The origin paths just include build/resources/main, not build/classes/main, but otherwise works great
-        FabricLoader.getInstance().getAllMods().stream().filter(modContainer -> modContainer.getOrigin().getKind() == ModOrigin.Kind.PATH).forEach(modContainer -> {
-            for(Path rootPath : modContainer.getOrigin().getPaths()) {
-                if(rootPath.toFile().equals(classFile)) {
-                    mods.add(new Mod(modContainer.getMetadata().getId(), modContainer.getMetadata()
-                            .getName(), modContainer.getMetadata().getVersion().getFriendlyString()));
-                }
-            }
-        });
+        FabricLoader.getInstance()
+                .getAllMods()
+                .stream()
+                .filter(modContainer -> modContainer.getOrigin().getKind() == ModOrigin.Kind.PATH)
+                .forEach(modContainer -> {
+                    for(Path rootPath : modContainer.getOrigin().getPaths()) {
```

Comment:
Does this list all paths, or just the entry points/root paths?
Asking because of the variable name?